models/stock_location.py

```diff
@@ -240,11 +241,12 @@ def _create_pi_count_move_picking(self, count_move, picking_type_id):
         elif 'quant_ids' in count_move:
             quant_ids = [int(x) for x in count_move['quant_ids']]
             quants = Quant.browse(quant_ids)
+            num_found_quants = len(list(filter(lambda q: q.exists(), quants)))
```

Comment:
Can we use len(quants.exists()) please? this will probably be more efficient since it is probably done in one SQL call instead of one per quant